resources/views/Listings/show.blade.php

```diff
@@ -0,0 +1,124 @@
+        <img src="{{$listing->getImageUrl()}}" alt="placeholder image">
+    </div>
+    <div class="right">
+        <h1>{{$listing->product->product_name}}</h1>
+        <p>{{$listing->product->description}}</p>
+        <p>Aangeboden door: {{$listing->user->name}} {{$listing->user->lastname}}</p>
+        @if ($listing->type == "bidding")
+            <p>Bieden vanaf: &euro;{{isset($listing->price_from) ? $listing->price_from : 0}}</p>
+            <div class="d-flex gap-3">
+                <p>Veiling loopt: </p>
+                <p id="timer"></p>
+            </div>
+            <div class="card mb-4">
+                <div class="card-body">
+                    @foreach ($listing->bids as $bid)
+                        <div class="d-flex flex-row gap-3">
+                            <p>{{$bid->date}}</p>
+                            <p>{{$bid->user->name}} {{$bid->user->lastname}}</p>
+                            <p>&euro;{{$bid->price}}</p>
+                        </div>
+                    @endforeach
+                </div>
+            </div>
+            <form method="POST" action="{{Route('listing.bid')}}">
+                @csrf
+                <input type="hidden" name="listing" value="{{$listing->id}}"/>
+                <div class="form-group mb-4">
+                    <input class="form-control border-black" name="bod" type="text" placeholder="0,00"/>
+                </div>
+                @error('bod')
+                    <div class="alert alert-danger mt-1 p-2">{{ $message }}</div>
+                @enderror
+                <div class="form-group mb-4">
+                    <input class="btn btn-primary" type="submit" value="Bod plaatsen" required/>
+                </div>
+            </form>
+        @elseif ($listing->type == "rental")
+            <form method="POST" action="{{Route('listing.rent')}}">
+                @csrf
+                <input type="hidden" name="listing" value="{{$listing->id}}"/>
+                <div class="form-group mb-4">
+                    <label for="rent_from">Start datum</label>
+                    <input class="form-control datepicker" type="text" name="rent_from"/>
+                </div>
+                <div class="form-group mb-4">
+                    <label for="rent_until">Eind datum</label>
+                    <input class="form-control datepicker" type="text" name="rent_until"/>
+                </div>
+                <div class="form-group mb-4">
+                    <input class="btn btn-primary" type="submit" value="Product Huren"/>
+                </div>
+            </form>
+        @else
+            <p>&euro;{{$listing->price}}</p>
+            <form method="POST" action="{{Route('listing.buy')}}">
+                @csrf
+                <input type="hidden" name="listing" value="{{$listing->id}}"/>
+                <input class="btn btn-primary" type="submit" value="Product Kopen"/>
+            </form>
+        @endif
+    </div>
+</div>
```

Comment:
Het fijn zou als er een knop is voor toevoegen aan favorieten bij het bekijken van het product. je hoeft alleen een knop of link toe te voegen en ik zal dan een route toevoegen

Reply:
Dit heb ik gedaan. Er zijn 2 hartjes iconen. Deze hoef je allene nog maar een if else op te doen en de route toevoegen.